addons/interact_menu/functions/fnc_renderIcon.sqf

```diff
@@ -33,14 +33,15 @@ if(GVAR(iconCount) > (count GVAR(iconCtrls))-1) then {
 };
 private _ctrl = GVAR(iconCtrls) select GVAR(iconCount);
 
-if(_icon == "") then {
-    _icon = DEFAULT_ICON;
+_icon params [["_iconFile", "", [""]], ["_iconColor", "#FFFFFF", [""]]];
+if (_iconFile isEqualTo "") then {
+    _iconFile = DEFAULT_ICON;
```

Comment:
Just put it in default value in `params` above?

Reply:
That would not work because `params` will not use `DEFAULT_ICON` if `_icon` is `""`.

Reply:
Ah, I see.